masonite/exception_handler.py

```diff
@@ -74,3 +75,6 @@ def _render(self):
                                                }
                                                ).rendered_template
         self._app.bind('Response', rendered_view)
+        request.header('Content-Type', str(len(rendered_view)))
+        self._app.bind('Headers', request.get_headers())
+        request.reset_headers()
```

Comment:
Why do we need to reset header and also why do we need request.get_headers, when you already set headers by request.header?